index.html

```diff
@@ -80,6 +80,12 @@
   });
 </script>
 
+  <script>
+      function handleClientLoad() {
+        gapi.load('client:auth2', function () { });
+      }
+  </script>
+
```

Comment:
I tried only loading this dependency when needed (on the import modals), but I couldn't get it to work.